src/page/Auth/FindPassword/SendAuthNumber/SendAuthNumber.module.scss

```
@@ -0,0 +1,74 @@
+.template {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 220px;
+  padding-bottom: 120px;
+  flex-direction: column;
+}
+
+.logo {
+  margin: auto;
+}
+
+.form {
+  width: 368px;
+  margin-top: 48px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  &__input {
+    width: 368px;
+    height: 48px;
+    padding: 14px 16px;
+    border: 1px solid #d2dae2;
+    color: #222;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+    box-sizing: border-box;
+  }
+
+  &__input--auth {
+    width: 255px;
+  }
+
```

```suggestion
    font-weight: 400;
    line-height: normal;
    box-sizing: border-box;

    &--auth{
    width: 255px;
    }
  }
```

`__input--auth`는 input안으로 넣어주셔도 될것같아요